### DISCUSSION TITLE
Control vibration in the Better xcloud App

### DISCUSSION THREAD

**Discussion by Maycaco39** [redphx/better-xcloud#341] [General]
My controller has access to vibration, I've already tested it to make sure, but the app says it doesn't support vibration.

**Comment by Jedibuji** [comment]
Similar issue here - got a new gamesir g8+ and the very first time I connected it and tested in BXcloud, I had rumble. The next time I connected it, it showed the red x on the vibration side of the connection message. I can't get it to turn back on, and have done all of the normal things like rebooting devices, resetting pairing, etc.
